### PR TITLE
Unit testing category

### PR DESCRIPTION
**Pull request by chaitanya005**
- Added Test cases for Category

### CODE REVIEW

**Comment by iMagesh** [comment]
@chaitanya005 can you resolve the conflict

**Comment by chaitanya005** [comment]
@iMagesh  resolved conflicts magesh please check it

File: client/src/components/Admin/Navbar.js [hunk 1]
```diff
@@ -11,14 +11,7 @@ const Navbar = () => {
         >
```
Comment (iMagesh) [inline client/src/components/Admin/Navbar.js:11]: you should have used a different branch for raising a new PR. Now see, both your PRs has the same set of changes. Always create new branch for every PR

Comment (chaitanya005) [reply client/src/components/Admin/Navbar.js:11]: sure @iMagesh I actually forgot that and commited the code, from next time I will create new branch for every PR.

**Pull request merged**